src/prettytable/prettytable.py

```diff
@@ -239,7 +242,6 @@ def __init__(self, field_names: Sequence[str] | None = None, **kwargs) -> None:
             "horizontal_align_char",
             "junction_char",
             "header_style",
-            "valign",
```

Comment:
`valign` is present twice in `self._options`.
https://github.com/jazzband/prettytable/blob/d80f5234a7dc27c72c1c080b6afb76ab131cc89a/src/prettytable/prettytable.py#L242-L255